### ISSUE TITLE
Env var NODE_ENV=development causes installation problems

### ISSUE THREAD

**Issue by RossVertizan** [appium/appium-desktop#1215]
## Appium or Appium Desktop?
- [ X ] I have verified that my issue does not occur with Appium and should be investigated as an Appium Desktop issue

## The problem
I downloaded and executed the file Appium-windows-1.15.1.exe, but the Appium desktop window did not launch.  On running it from a command prompt, I could see that it failed with an error stating that it could not find the module "electron-devtools-installer".

I was able to get past that problem by navigating to the app directory and running a "yarn install electron-devtools-installer".  This allowed the desktop to launch, interestingly it launched in dev mode with the Chrome console etc. visible.  When I launched the inspector, the inspector window came up, but it never managed to load the screenshot of the phone, it just sat with the circling dots.  The console showed an error, indicating that there was not an active session.

I had had problems previously which I filed as user Graar ( see #554 ). On that occasion @amarchbanks commented on the use of the environment variable NODE_ENV when it is set to development.  As I am developing an app, I have this env var set to "development", so I deleted it from my environment and uninstalled and reinstalled and voila ... everything works as it should.

So, somewhere in the build process, there is a remaining dependency on that env var.

I hope this helps someone.

## Environment

- I am running Appium Desktop version _1.15.1_.
- I am on (pick one):
    - [ ] Mac
    - [ X ] Windows
    - [ ] Linux

**Comment by KazuCocoa** [comment]
ah...

thanks. perhaps we can add the environment variable in https://github.com/appium/appium-desktop#advanced-topics-and-troubleshooting

**Comment by RossVertizan** [comment]
Yes, that is a good short term solution.  However, in the longer term maybe look at the build scripts that are used to build the installer to remove the dependency on the NODE_ENV env var.
